internal/lookout/ui/src/services/ComputeResourcesService.ts

```diff
@@ -0,0 +1,89 @@
+import { load } from "js-yaml"
+
+// A simple interface that our PodSpec has.
+interface JobYaml {
+  podSpec: {
+    containers: Container[]
+  }
+}
+interface Container {
+  command: string[]
+  args: string[]
+  resources: {
+    limits: Limits
+  }
+}
+
+interface Limits {
+  cpu?: string
+  "nvidia.com/gpu"?: string
+  storage?: string
```

Comment:
This should be `ephemeral-storage` (or `ephemeralStorage`, worth double checking what the API returns)

Reply:
Good call. Changed to ephemeral-storage.